dss/api/bundles/__init__.py

```diff
@@ -52,7 +52,7 @@ def get(
 
     if directurls or presignedurls:
         try:
-            token, ready = verify_checkout(_replica, uuid, version, token)
+            token, ready = verify_checkout(_replica, uuid, version, token, bundle_metadata)
```

Comment:
Does this code get copied over to the step function/daemon code during deployment?

Context: Passing in `bundle_metadata` here to reduce calls to `get_bundle_manifest`. Potentially a better solution pointed out by @kislyuk would be to use the `@functools.lru_cache` decorator on `get_bundle_manifest`.